examples/map_text_with_nomic_embed.py

```diff
@@ -41,5 +41,5 @@ def generate_embeddings(documents):
 document_embeddings = generate_embeddings(documents)
 
 print(document_embeddings.shape)
-response = atlas.map_embeddings(embeddings=document_embeddings, data=documents, topic_label_field='text', build_topic_model=True)
+response = atlas.map_data(embeddings=document_embeddings, data=documents, topic_model=dict(topic_label_field='text'))
```

Comment:
```suggestion
response = atlas.map_data(embeddings=document_embeddings, data=documents, topic_model={"topic_label_field" : "text"}))
```

Reply:
Putting `topic_label_field` as an argument inside `topic_model` feels redundant--should possibly be just `label_field`.